testsuite/exp

```diff
@@ -214,7 +213,7 @@ def workflow_dispatch_forge(
         "--field",
         f"FORGE_TEST_SUITE={test_suite}",
         "--field",
-        f"FORGE_CLUSTER_NAME={cluster_name}",
+        f"FORGE_CLUSTER_NAME={FORGE_DEFAULT_CLUSTER_NAME}",
```

Comment:
Can we make this an argument instead?

Reply:
sure, done. I expect most will not set this though, as we want folks to use the dedicated experimental cluster. This might be useful for the the multi-region karmada forge that @ibalajiarun runs though.

Reply:
Nice, you can also do it in a future PR to unblock this now